logs/events.log

```diff
@@ -1,2 +0,0 @@
-2020-05-06 13:32:59 INFO  eventsLogger: Extrnal system added
-2020-05-06 13:49:44 INFO  eventsLogger: Extrnal system added
```

Comment:
maybe add events.log to gitignore while the system is still developing? to prevent conflicts (log will show different raws on my pc then on yours etc...)